client/common/components/BoardSection.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Task from '../types/Task';
+import Card from './Card';
+
+interface BoardInterface {
+  title: String;
```

Comment:
Is this interface going in the future for the types folder ?

Reply:
I'll move them all to interfaces so the code is cleaner :) The types for the props (you'll see that in the next commit) have to stay on the same page though.